charts/trino/templates/service-coordinator.yaml

```
@@ -1,4 +1,5 @@
 {{- $coordinatorJmx := merge .Values.jmx.coordinator (omit .Values.jmx "coordinator" "worker") -}}
+{{- $svc := .Values.service | default .Values.serviceCoordinator }}
```

Should we merge the coordinator service with the "general" service, like we do for JMX above?

---

IMHO it's better to keep them separated, or even add a deprecation warning for `Values.service`.

---

I'm ok with keeping them separated, but we should make sure no values are silently ignored. So we should call `fail()` when using the coordinator or worker services, and there's anything set in the shared service.